keras_cv/visualization/plot_image_gallery.py

```diff
@@ -117,9 +117,9 @@ def plot_image_gallery(
     """
     assert_matplotlib_installed("plot_bounding_box_gallery")
 
-    if path is None and show is None:
-        # Default to showing the image
-        show = True
+    if path is None and not show:
+        plt.ioff()
```

Comment:
Why do we want to make this non-interactive?

Reply:
Sure, won't remove the interactive mode